### PR TITLE
Business page next

### PR DESCRIPTION
**Pull request by madsobel**

### CODE REVIEW

File: client/components/Map/__snapshots__/MapPin.spec.jsx.snap [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MapPin Snapshots should match the snapshot 1`] = `
+<svg
```
Comment (andersklenke) [inline client/components/Map/__snapshots__/MapPin.spec.jsx.snap]: I would be nice to mock out SVG's in the specs so we avoid this in the snapshot.

**Comment by andersklenke** [comment]
Can you also add a Storybook for `Map`?

File: client/components/Map/__snapshots__/MapPin.spec.jsx.snap (hunk 1 above)
Comment (madsobel) [reply client/components/Map/__snapshots__/MapPin.spec.jsx.snap]: How would imagine that the snapshot then would look?
This is how the rendered SVG would look like in a browser.

**Comment by madsobel** [comment]
@andersklenke Looked into Storybook, and it actually seems like it's broken after moving to Next.js.
I think it's a better idea to look into this in a separate PR

**Pull request merged**